QUARKUS-2479.md

```diff
@@ -0,0 +1,31 @@
+# QUARKUS-2479 SmallRye Config SecretKeys support
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2479
+
+This test plan covers support of secret keys in SmallRye config
+
+## Scope of the testing
+Following topics should be covered:
+
+1. Verify, that some config properties can be marked as secret.
+2. Verify, that usage of these secrets leads to an exception.
+3. Verify, that these secrets cannot be used when injected and when accessed directly.
+4. Verify, that these secrets can be used only within doUnlocked section.
+5. Coverage should exist for both JVM and native mode in baremetal.
+
+### Impact on test suites and testing automation
+- One new test class added for Kafka Dev Services
+- One test method in sql/hibernate reactive refactored 
+
+### Impact on resources:
+- No additional requirements for resources in lab
+- Required additional time for the test execution in JVM mode should be less, than minute.
```

Comment:
`less, than` ... why is the comma here?

You probably missed that in my suggestion https://github.com/quarkus-qe/quarkus-test-plans/pull/105#discussion_r1005492654